test/mocha/10-api.js

```diff
@@ -86,14 +86,14 @@ describe('bedrock-account-http', function bedrockAccountHttp() {
       result.status.should.equal(201);
     });
 
-    it('should return 201 for accounts with the same email', async function() {
+    it('should return 409 for accounts with the same email', async function() {
       const email = {email: 'multiple@digitalbazaar.com'};
       const result1 = await api.post('/', email);
       const result2 = await api.post('/', email);
-      const result3 = await api.post('/', email);
+
       result1.status.should.equal(201);
-      result2.status.should.equal(201);
-      result3.status.should.equal(201);
+      result2.data.message.should.equal('Duplicate account.');
```

Comment:
There's a property that contains `DuplicateError` is that correct, if so, please add assertion for that.

Reply:
Updated here 6039e4b